src/map/atcommand.c

```diff
@@ -9813,6 +9813,8 @@ bool atcommand_exec(const int fd, struct map_session_data *sd, const char *messa
 			clif->message(fd, msg_fd(fd,143));
 			return false;
 		}
+		if ( sd->block_action & PCBLOCK_COMMAND ) // *pcblock script command
```

Comment:
In cases like this that are a response to a direct action from the player, I think we should show an informative message explaining why they can't do it (similarly to what happens with the nocommand mapflag).

PCBLOCK_SITSTAND and possibly some others, would be nice if they had similar messages.